src/en/juju-offline-charms.md

```diff
@@ -1,124 +1,94 @@
-reasons.
+# Deploying charms offline
 
-In these cases it's useful to maintain a copy of the internet-accessible
-Charm Store for your local deployments.
+There are times when it may not be possible to use the charms located in the
+official Charm Store. Such cases include:
 
-!!! Note: Though this method will ensure that the charms themselves are
-available on systems without outside internet access, there is no
-guarantee that a charm will work in a disconnected state. Some charms
-pull code from the outside world, such as github. We recommend modifying
-these charms to pull code from an internal server when appropriate.
+- The backing cloud may be private and not have internet access.
+- The charms may not exist online. They are newly-written charms.
+- The charms may exist online but they have been customized.
 
-## Retrieving charms using the Charm Tools
+This page will explain how to manage local Juju charms.
 
-### Installation
+!!! Note: Although this method will ensure that the charms themselves are
+available on systems without outside internet access, there is no guarantee
+that a charm will work in a disconnected state. Some charms pull code from the
+internet, such as GitHub. We recommend modifying these charms to pull code from
+an internal server instead.
 
-In addition to [Juju](getting-started#installation) we need
-to install charm-tools:
 
-```bash
-sudo apt-get update && sudo apt-get install charm-tools
-```
+## Using Charm Tools
 
-### Usage
+Charm Tools is a set of tools necessary when using Juju with locally stored charms.
 
-The Charm Tools comes packaged as both a stand alone tool and a juju plugin.
-So you simply can call it with `charm` or as usual for Juju commands with
-`juju charm`.
+See [Charm Tools](https://jujucharms.com/docs/devel/tools-charm-tools) for more
+information.
 
-There are several tools available within the Charm Tools itself. At any time
-you can run `juju charm` to view the available subcommands and all subcommands
-have independent help pages, accessible using either the `-h` or `--help` flags.
+### Installation
 
-If you want to retrieve and branch one of the charm store charms, use the `get`
-command specifying the `CHARM_NAME` you want to copy and provide an optional
-`CHARMS_DIRECTORY`. Otherwise the current directory will be used.
+Users of Ubuntu 14.04 (Trusty) will need to first add a PPA:
 
 ```bash
-juju charm get [-h|--help] CHARM_NAME [CHARMS_DIRECTORY]
+sudo add-apt-repository ppa:juju/stable
+sudo apt update
 ```
 
-### Example
-
-The command
+Install the software:
 
 ```bash
-juju charm get mysql
+sudo apt install charm-tools
 ```
 
-will download the MySQL charm to a `mysql` directory within your current path.
-By running
+### Usage
 
-```bash
-juju charm get wordpress ~/charms/precise/
-```
+These tools come packaged as a standalone tool and should no longer be used as
+a Juju plugin. The commands are therefore called with `charm <subcommand>`.
```

Comment:
I realize you are giving a road sign to users of Juju 1.x. I wonder if it is necessary. Consider replacing lines 55-56 with this:
Charm commands are called with `charm <subcommand>`.